src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,15 +1,17 @@
 {{#> page-header-tools}}
   {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-icons"}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Alerts"'}}
+    {{#> button button--modifier="pf-m-plain pf-c-page__header-notifications" button--attribute='aria-label="Unread notifications"'}}
```

Comment:
@mattnolting thanks for that! I merged in your suggestions, do you think its ok that we add these classes: `pf-c-page__header-notifications` and `pf-c-page__header-settings` to the button modifier and `pf-c-page__header-overflow-menu` to the dropdown modifier. I was tossing up between doing this also because i'm not sure if it follows what we typically do. @mcoker what do you think?

Reply:
Yeah, probably should wrap those. Looks a lot cleaner in the code too.

Reply:
by wrap, do you mean create elements in the page header where we place the buttons? If so, that's what I would do, too.